pages/docs/community/index.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
of course you remembered to add the `overview` page! i LOVE YOU, you are sooo good at paying attention to the little details! ✨😄